backend/controller/ingress/handler.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/TBD54566975/ftl/internal/model"
 )
 
+const internalServerError = "Internal Server Error"
```

Comment:
An open question is if we want to include this message or just an empty string.

Reply:
```suggestion
const internalServerError = http.StatusText(http.StatusInternalServerError)
```

(or inline that call elsewhere)

Reply:
I don't think that compiles, I would need to inline it

Reply:
Although it did make me realize I am sending the wrong message in some places

Reply:
Make it `var ..` to compile.

Reply:
I inlined it as some of the messages were for different response codes